pom.xml

```diff
@@ -95,6 +95,7 @@
     <ant.version>1.9.1</ant.version>
     <antlr.version>3.4</antlr.version>
     <avro.version>1.7.5</avro.version>
+    <avroHelper.version>0.2.48</avroHelper.version>
```

Comment:
Can you use a more recent version? 0.2.71 is the latest release.

Reply:
Added latest avro-util version 0.2.71.